### PR TITLE
v2ray: update sed args and upx ignore errors

### PR DESCRIPTION
**Pull request by Mattraks**
**v2ray: update sed args and upx ignore errors**

### CODE REVIEW

File: package/lean/v2ray/Makefile [hunk 1]
```diff
@@ -271,7 +282,7 @@ define Build/Compile
 	mv -f $(GO_PKG_BUILD_BIN_DIR)/main $(GO_PKG_BUILD_BIN_DIR)/v2ray
 
 ifeq ($(CONFIG_V2RAY_COMPRESS_UPX),y)
-	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/v2ray
+	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/v2ray || true
```
Comment (1715173329) [inline package/lean/v2ray/Makefile:285]: What's the point of this change?

File: package/lean/v2ray/Makefile [hunk 2]
```diff
@@ -280,7 +291,7 @@ ifneq ($(CONFIG_V2RAY_EXCLUDE_V2CTL),y)
 	mv -f $(GO_PKG_BUILD_BIN_DIR)/main $(GO_PKG_BUILD_BIN_DIR)/v2ctl
 
 ifeq ($(CONFIG_V2RAY_COMPRESS_UPX),y)
-	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/v2ctl
+	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/v2ctl || true
```
Comment (1715173329) [inline package/lean/v2ray/Makefile:294]: Same.

**Review by 1715173329 — COMMENTED** [review]
Please bump PKG_RELEASE

File: package/lean/v2ray/Makefile (hunk 1 above)
Comment (Mattraks) [reply package/lean/v2ray/Makefile:285]: upx ignore errors Continue to compile

File: package/lean/v2ray/Makefile (hunk 2 above)
Comment (Mattraks) [reply package/lean/v2ray/Makefile:294]: Ignore the error and continue to compile

Comment (1715173329) [reply package/lean/v2ray/Makefile:294]: > Ignore the error and continue to compile

So is there any example? I don't ever see any error with existing command.

Comment (Mattraks) [reply package/lean/v2ray/Makefile:294]: For example, upx is missing or compression error, this is only to prevent upx error

Comment (1715173329) [reply package/lean/v2ray/Makefile:294]: All right, please bump PKG_RELEASE.

**Review by 1715173329 — APPROVED** [review]

File: package/lean/v2ray/Makefile (hunk 2 above)
Comment (Mattraks) [reply package/lean/v2ray/Makefile:294]: updated

**Comment by 1715173329** [comment]
Thanks for PR.

**Pull request merged**